opensrp-chw-anc/src/main/java/org/smartregister/chw/anc/util/JsonFormUtils.java

```diff
@@ -218,6 +223,30 @@ public static String getValue(JSONObject jsonObject, String key) {
         return "";
     }
 
+    /**
+     * Returns an Array value from JSON form field
+     *
+     * @param jsonObject native forms jsonObject
+     * @param key        field object key
+     * @return Array value
+     */
+    public static JSONArray getValueArray(JSONObject jsonObject, String key) {
```

Comment:
Is this method meant to only work with single step form?

Reply:
Refactored to handle a single step that's specified by passing the step name as a param ✅